tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+class ActionSelenium(argparse.Action):
+    """
+    Validate the comma-separated list of requested browsers.
+    """
+    def __call__(self, parser, namespace, values, option_string=None):
+        browsers = values.split(',')
+        for browser in browsers:
+            try:
+                # Attempt to create class instance to see if everything is fine.
```

Comment:
Seems like this comment can be removed since it's obsolete and the code is simple now.